src/orchestrator.ts

```diff
@@ -251,12 +253,16 @@ export class Orchestrator {
     }
 
     private callSubOrchestrator(state: HistoryEvent[], name: string, input?: unknown, instanceId?: string): Task {
+        if (!name) {
+            throw new Error("A suborchestration function name must be provided when attempting to create a suborchestration");
```

Comment:
nit: in the documentation we use sub-orchestration with the hyphen. We should probably be consistent with this spelling in our error messages as well.